js-weather/index.js

```diff
@@ -0,0 +1,50 @@
+var API_KEY = '4b5774e9f3d2a07b84f0f2f88e486224';
```

Comment:
```suggestion
const API_KEY = '4b5774e9f3d2a07b84f0f2f88e486224';
```

変数は全て const で宣言しましょう。varは廃止します。

Reply:
3b93282
こちらのコミットで対応しました。

他の変更も同時にコミットしており申し訳ありません。以後注意します。